### PR TITLE
[fr]: Update some French values

### PR DESCRIPTION
**Pull request by dizco**
- "Pas content" in French is "Not happy". "Pas de contenu" is more appropriate
- OK is accepted in French
- "Erreur de canon ferroviaire" essentially means "error of the railroad gun", doesn't make sense. Railgun is an actual name of a Cloudflare product, its more appropriate to keep it as-is
- "Demander des champs d'en-tête trop grands" is like "ask for bigger fields". "Champs d'en-tête de requête trop grands" feels more appropriate

### CODE REVIEW

File: locales/fr/json.json [hunk 1]
```diff
@@ -35,14 +35,14 @@
...
-    "No Content": "Pas content",
+    "No Content": "Pas de contenu",
     "Non-Authoritative Information": "Informations non autorisées",
     "Not Acceptable": "Pas acceptable",
     "Not Extended": "Non prolongé",
     "Not Found": "Page non trouvée",
     "Not Implemented": "Pas mis en œuvre",
     "Not Modified": "Non modifié",
-    "OK": "D'accord",
+    "OK": "OK",
```
Comment (andrey-helldar) [inline locales/fr/json.json:45]: In this case, you will need to create a `_excludes.json` file and add a value to it:

```json
[
    "OK"
]
```

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/fr/json.json (hunk 1 above)
Comment (dizco) [reply locales/fr/json.json:45]: Updated

**Comment by caouecs** [comment]
c'est bon pour moi

it's ok

**Review by andrey-helldar — APPROVED** [review]
Merci

**Pull request merged**